include/deal.II/matrix_free/operators.h

```diff
@@ -174,6 +174,12 @@ namespace MatrixFreeOperators
   protected:
 
     /**
+     * Set constrained entries (both from hanging nodes and edge constraints)
+     * of @p dst to unity.
+     */
+    void set_constrained_entries_to_unity (LinearAlgebra::distributed::Vector<Number> &dst) const;
```

Comment:
I also agree that this is a better name. But I see that @davydden has already taken that one :+1: